### ISSUE TITLE
Errors after opening `index.html` from `romcal-es6`

### ISSUE THREAD

**Issue by tukusejssirs** [romcal/romcal#55]
I opened `index.html` from `romcal-es6` branch in Firefox. Then I opened the web console and I have found the following errors:

```
Error: Cannot find module "fs" romcal.bundle.min.js line 2097 > eval:1:227

    webpackMissingModule webpack:///./node_modules/@std/esm/index.js?:1 <anonymous> webpack:///./node_modules/@std/esm/index.js?:1 <anonymous> webpack:///./node_modules/@std/esm/index.js?:1 js file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:2097 __webpack_require__ file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:20 <anonymous> webpack:///./index.js?:1 <anonymous> webpack:///./index.js?:1 js file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:2072 __webpack_require__ file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:20 <anonymous> webpack:///multi_./index.js?:1 0 file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:2109 __webpack_require__ file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:20 <anonymous> file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:69 <anonymous> file:///home/ts/git/romcal-es6_2/dist/romcal.bundle.min.js:1 

ReferenceError: Romcal is not defined[Learn More] index.html:33:11

    <anonymous> file:///home/ts/git/romcal-es6_2/index.html:33 

The Components object is deprecated. It will soon be removed. index.html
```

There are many issues, although ATM I have no knowledge what is going on or how can I solve it.

**Comment by tukusejssirs** [comment]
When doing the same produre after [building it](https://github.com/pejulian/romcal/issues/49#issuecomment-462069623), the errors are as follows:

```
ReferenceError: exports is not defined[Learn More] romcal.bundle.min.js line 98 > eval:4:1

    <anonymous> webpack:///./src/index.mjs?:4 mjs file:///home/ts/git/romcal-es6/dist/romcal.bundle.min.js:98 __webpack_require__ file:///home/ts/git/romcal-es6/dist/romcal.bundle.min.js:20 <anonymous> webpack:///multi_./src/index.mjs?:1 0 file:///home/ts/git/romcal-es6/dist/romcal.bundle.min.js:110 __webpack_require__ file:///home/ts/git/romcal-es6/dist/romcal.bundle.min.js:20 <anonymous> file:///home/ts/git/romcal-es6/dist/romcal.bundle.min.js:84 <anonymous> file:///home/ts/git/romcal-es6/dist/romcal.bundle.min.js:1 

ReferenceError: Romcal is not defined[Learn More] index.html:33:11

    <anonymous> file:///home/ts/git/romcal-es6/index.html:33 

The Components object is deprecated. It will soon be removed. index.html
```

**Comment by tukusejssirs** [comment]
According the description of the [milestone 1](https://github.com/romcal/romcal/milestone/1), `romcal-es6` supports only `drops support for Node v4, v5 while adding support for Node v8 and v9`, which actually means `>=6.0.0`, as it is written in `package.json` on [line 6](https://github.com/romcal/romcal/blob/217efca8ae80a45a04297ab1ede38844c76e36a0/package.json#L8). Thereforr _should_ work with node v10.14.1, as triaged by @jarosz, however there might some changes in node v9+.

**Comment by tukusejssirs** [comment]
Consider using [Greenkeeper](https://greenkeeper.io/), as suggested [here](https://github.com/romcal/romcal/issues/54#issuecomment-461996319).

**Comment by emagnier** [comment]
Fixed here #82
